OpenRA.Platforms.Default/ThreadedGraphicsContext.cs

```diff
@@ -683,6 +713,11 @@ public ThreadedShader(ThreadedGraphicsContext device, IShader shader)
 			setVec4 = tuple => { var t = ((string, float, float, float))tuple; shader.SetVec(t.Item1, t.Item2, t.Item3, t.Item4); };
 		}
 
+		public void Bind()
```

Comment:
Expensive. To post this as a separate message. 

Isn't the current use often that Vertices are created/initialized each render. (Even they come from the pool). In this case, wouldn't an auto bind at initial creation be an option?

Reply:
It's not an option as the renderer decides what to render independent of vertex type. But I do plan to add bind caching.

As we never unbind, we could have static variables to make sure we never bind the same class twice